examples/text_classification/bert/bert_handler.py

```diff
@@ -67,12 +89,16 @@ def inference(self, input_ids, topk=5):
 
         self.model.eval()
         outputs = self.model(input_ids)
+        output = outputs[0].argmax(1).item() + 1
```

Comment:
@HamidShojanazeri Please add error handling condition for scenarios when there is no matching output returned by the model